oneview/resource_server_profile_template.go

```diff
@@ -139,22 +158,31 @@ func resourceServerProfileTemplate() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
 							Computed: true,
+							//							iDiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+							//								c := []byte("dummy")
+							//								_ = ioutil.WriteFile(old, c, 0644)
+							//								_ = ioutil.WriteFile(new, c, 0644)
+							//								_ = ioutil.WriteFile(k, c, 0644)
+							//								return false
+							//							},
 						},
 						"connections": {
 							Optional: true,
 							Computed: true,
-							Type:     schema.TypeList,
+							Type:     schema.TypeSet,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									"boot": {
 										Optional: true,
 										Computed: true,
 										Type:     schema.TypeList,
+										MaxItems: 1,
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
 												"boot_vlan_id": {
-													Type:     schema.TypeInt,
+													Type:     schema.TypeString,
 													Optional: true,
+													Computed: true,
```

Comment:
why computed is added here? should not be only optional. If I can see boot_volume_soure is only set to optional

Reply:
will need to take care of this in next PR